### ISSUE TITLE
Priority for manning balista when hovering over crowded tower

### ISSUE THREAD

**Issue by an-accident** [Sh0wdown/UnofficialCrusaderPatch#545]
**What happens**
It's very hard to send engineers to the balista if there are lot of troops in the tower. Often pixel-hunting for the right spot to click or even sending all troops out for a moment is required.
**What would be QoL improvement in this situation**
When engineers are selected action for manning stuff should take priority over action for selecting troops, i.e. arrows for <man that balista> should show instead of circle for <selecting troops>.

**Comment by GRhin** [comment]
Would this be solved / be an extension of the idea posted in #211 where Using the "shift" key would cause you to give an instruction to selected units instead of creating a new selection?

**Comment by Monsterfisch** [comment]
yes it would as that sort of problem was what i had in mind when making the OP
